src/modules/tests/test-data/cat-c/test-data.cat-c.reducer.ts

```diff
@@ -0,0 +1,43 @@
+import { CatCUniqueTypes } from '@dvsa/mes-test-schema/categories/C';
+import { createFeatureSelector, combineReducers, Action } from '@ngrx/store';
+
+import { dangerousFaultsReducer } from '../common/dangerous-faults/dangerous-faults.reducer';
+import { drivingFaultsReducer } from '../common/driving-faults/driving-faults.reducer';
+import { ecoReducer } from '../common/eco/eco.reducer';
+import { etaReducer } from '../common/eta/eta.reducer';
+import { manoeuvresCatBEReducer } from './manoeuvres/manoeuvres.cat-c.reducer';
+import { seriousFaultsReducer } from '../common/serious-faults/serious-faults.reducer';
+import { vehicleChecksCatBEReducer } from './vehicle-checks/vehicle-checks.cat-c.reducer';
+import { testRequirementsCatBEReducer } from './test-requirements/test-requirements.cat-c.reducer';
+
+export const initialState: CatCUniqueTypes.TestData = {
+  dangerousFaults: {},
+  drivingFaults: {},
+  manoeuvres: {},
+  seriousFaults: {},
+  testRequirements: {},
+  ETA: {},
+  eco: {},
+  vehicleChecks: {
+    tellMeQuestions: [],
+    showMeQuestions: [],
+  },
+};
+
+export function testDataCatCReducer(
+  state: CatCUniqueTypes.TestData,
+  action: Action,
+): CatCUniqueTypes.TestData {
+  return combineReducers({
+    drivingFaults: drivingFaultsReducer,
+    dangerousFaults: dangerousFaultsReducer,
+    seriousFaults: seriousFaultsReducer,
+    vehicleChecks: vehicleChecksCatBEReducer,
+    eco: ecoReducer,
+    ETA: etaReducer,
+    manoeuvres: manoeuvresCatBEReducer,
+    testRequirements: testRequirementsCatBEReducer,
+  })(state as Required<CatCUniqueTypes.TestData>, action);
+}
+
+export const getTestData = createFeatureSelector<CatCUniqueTypes.TestData>('testData');
```

Comment:
CatBE reducer name  in cat c reducer files.

Reply:
changed it